src/components/Form/Checkbox.jsx

```diff
@@ -21,33 +22,42 @@ const StyledCheckbox = styled.input.attrs({
   margin: 0;
 
   & + label {
-    font-size: ${props => props.theme.form.label.fontSize};
-    line-height: ${props => props.theme.form.label.fontSize};
+    font-size: ${props => props.theme.form.checkbox[props.size].size};
+    line-height: ${props => props.theme.form.checkbox[props.size].size};
     font-weight: ${props => props.theme.form.label.fontWeight};
-    padding-left: 2.5rem;
+    padding-left: ${props => props.theme.form.checkbox[props.size].paddingLeft};
     cursor: pointer;
+    transition: opacity 0.2s ease-in-out;
 
     &:before,
     &:after {
       display: inline-block;
       position: absolute;
       left: 0;
-      top: 0.4rem;
+      top: ${props => props.theme.form.radio[props.size].top};
       border: 1px solid #ccc;
       border-radius: 3px;
       border-color: ${props => props.theme.form.checkbox.borderColor};
       content: '';
-      width: 1.7rem;
-      height: 1.7rem;
-      transition: border 0.15s ease-in-out, color 0.15s ease-in-out;
+      width: ${props => props.theme.form.radio[props.size].size};
+      height: ${props => props.theme.form.radio[props.size].size};
+    }
+
+    &:after {
```

Comment:
You should "prefix" pseudo elements with `::`, not just one. It is a way to distinguish pseudo elements from pseudo classes.

Reply:
Done.